lib/models/image_layer.js

```diff
@@ -11,6 +11,7 @@ export const DEFAULT_IMAGE_LAYER = {
   name:                "[Image]",
   type:                IMAGE_TYPE,
   renderOrder:         0,
+  hide:                false,
```

Comment:
Been thinking about it, and I think we need to go with `visible: true` for this property name. (Please don't shoot me!)

Reply:
I hate you a bit, but it's fine XD. Hahaha, changed it. I only did hide because of backwards compatibility, but since that's handled as per below comment, it makes way more sense to be visible. Changed it.